html/changelogs/burgerbb-chainsaw.yml

```diff
@@ -0,0 +1,4 @@
+author: BurgerBB
+delete-after: True
+changes: 
+  - rscadd: "Added chainsaws. Chainsaws are two handed weapons that require welder fuel to operate. Two hands a required to turn the chainsaw on. Chainsaws can open airlocks and lockers."
```

Comment:
> Two hands a required to turn the chainsaw on. 

That is redundant. Its mentioned in the second sentence that chainsaws are two-handed.
Also, let them fiure out the part about airlocks and lockers themselfs (We dont need to give everything away)